src/Compilers/CSharp/Portable/CodeGen/CodeGenerator.cs

```diff
@@ -348,6 +348,11 @@ private void EmitSymbolToken(FieldSymbol symbol, SyntaxNode syntaxNode)
             _builder.EmitToken(_module.Translate(symbol, syntaxNode, _diagnostics), syntaxNode, _diagnostics);
         }
 
+        private void EmitFunctionPointerSymbolToken(FunctionPointerTypeSymbol symbol, SyntaxNode syntaxNode)
```

Comment:
>EmitFunctionPointerSymbolToken [](start = 21, length = 30)

Consider using "EmitSignatureToken" as the name. For me "FunctionPointer" still sounds like this is about a type, i.e. about a type token.